pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -1163,6 +1151,32 @@ func (l *LibvirtDomainManager) findOrCreateVirDomain(
 	return virDomain, err
 }
 
+func (l *LibvirtDomainManager) handleNetworkAttachement(
+	apiDomain *api.Domain,
+	virDomain cli.VirDomain,
+	vmi *v1.VirtualMachineInstance,
+	options *cmdv1.VirtualMachineOptions,
+	oldSpec *api.DomainSpec,
```

Comment:
nit: I would place the domain related args close to each other.

Reply:
Ok